README.md

```diff
@@ -3,6 +3,7 @@
 [![Latest Version](https://img.shields.io/pypi/v/amazon-braket-default-simulator.svg)](https://pypi.python.org/pypi/amazon-braket-default-simulator)
 [![Supported Python Versions](https://img.shields.io/pypi/pyversions/amazon-braket-default-simulator.svg)](https://pypi.python.org/pypi/amazon-braket-default-simulator)
 [![Build Status](https://img.shields.io/github/workflow/status/aws/amazon-braket-default-simulator-python/Python%20package/main?logo=github)](https://github.com/aws/amazon-braket-default-simulator-python/actions?query=workflow%3A%22Python+package%22)
+[![codecov](https://codecov.io/gh/aws/amazon-braket-default-simulator-python/branch/main/graph/badge.svg?token=FZEG1AQU3S)](https://codecov.io/gh/aws/amazon-braket-default-simulator-python)
```

Comment:
Why codecov instead of Code Coverage to match the case and style of the rest?

Reply:
Just copied this badge directly from our Codecov page; figured it was most idiomatic for the service